src/common/dsprand.h

```diff
@@ -32,13 +39,98 @@ class WELL512
         return state[index];
     }
 
-    // generates double floating point numbers in the half-open interval [0, 1)
```

Comment:
stick with using interval notation for the comments, it is actually *more* readable than spelling it out (e.g. between and inclusive zero and the specified max value)